docker/bin/zookeeperStart.sh

```diff
@@ -99,9 +98,35 @@ if [[ "$WRITE_CONFIGURATION" == true ]]; then
 
     echo Registering node and writing local configuration to disk.
     java -Dlog4j.configuration=file:"$LOG4J_CONF" -jar /root/zu.jar add $ZKURL $MYID  $ZKCONFIG $DYNCONFIG
-
   fi
 fi
 
+ZOOCFGDIR=/data/conf
+export ZOOCFGDIR
+
+if [ ! -d "$ZOOCFGDIR" ]
+then
+echo "Creating writable conf directory."
```

Comment:
I feel like we should make a comment here to make the reasoning for this more explicit:

```
# Copying configmap generated contents to writable directory, to support Zookeeper dynamic reconfiguraiton
```

Or some such.

Reply:
Sure, will add more comments.